hack/cluster.sh

```diff
@@ -90,14 +96,24 @@ main() {
 	local op="$1"
 	shift
 	cd "$PROJECT_ROOT"
-
-	# NOTE: all operations are relative to tmp
+	export PATH="$BIN_DIR:$PATH"
 	mkdir -p "${TMP_DIR}"
-	git_checkout
+	ensure_all_tools
+
+	#NOTE: allow cluster_<OP> to executed without going through the entire
+	# setup again. This is useful (especially) in CI which uses the kepler-action
+	# to setup a cluster which does not have all the prerequisites for running
+	# Operator as OLM Bundle
+	declare -F | cut -f3 -d ' ' | grep -qE "^cluster_$op\$" && {
+		header "Running cluster_$op"
+		"cluster_$op" "$@"
```

Comment:
Is there any need to pass args to cluster_prereqs?

Reply:
this applies to all `cluster_op` so it is best to pass all args to that function .

Reply:
We only have cluster_prereqs and we don't use args there. What's the point of passing args there?